languages/javascript/scripts/scaffold/template/.meta/config.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Just in case this might be commited by accident, let's remove the `---`. Empty is a bit easier to handle.